<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plotPropertiesGUI</class>
 <widget class="QDialog" name="plotPropertiesGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Properties</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="topMargin">
          <number>8</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Border color:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="borderColorButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
&lt;!-- border-radius: 4px; 
  border: 1px solid rgb(0, 0, 0); 
  background-color:white; --&gt;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Background color:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="bckgrndColorButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
  border-radius: 4px; 
  border: 1px solid rgb(0, 0, 0); 
  background-color:white; 
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Canvas color:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="canvasColorButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
  border-radius: 4px; 
  border: 1px solid rgb(0, 0, 0); 
  background-color:white; 
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="topMargin">
          <number>8</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Border width: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="borderWidthSpinBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Margin: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="marginSpinBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="fontGroupBox">
         <property name="title">
          <string>Fonts </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
          <item>
           <widget class="QPushButton" name="titlesFontButton">
            <property name="text">
             <string>Titles</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="axesFontButton">
            <property name="text">
             <string>Axes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="legendsFontButton">
            <property name="text">
             <string>Legends</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="curvesTab">
      <attribute name="title">
       <string>Curves</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="treeCurve">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Curves</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QFrame" name="frame">
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="3">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Symbol size:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="curveStyleCombo">
              <item>
               <property name="text">
                <string>No curve</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Lines</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Sticks</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Steps</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Dots</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Line:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="lineStylecombo">
              <item>
               <property name="text">
                <string>No line</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>_____</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>- - - - - </string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>. . . . .</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>- · - · - </string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>- . . - . . </string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="2">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>6</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Symbol:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="4">
             <widget class="QSpinBox" name="symbolSizeSpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Color:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="curveSymbolCombo">
              <item>
               <property name="text">
                <string>No symbol</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ellipse</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Rect</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Diamond</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Triangle</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Downwards triangle</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Upwards triangle</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Left triangle</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Right triangle</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Cross</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Vertical cross</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Horizontal line</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Vertical line</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Asterisk</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Star</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Hexagon</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QSpinBox" name="curveWidthSpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Curve width:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Style:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QPushButton" name="curveColorButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>22</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
  border-radius: 4px; 
  border: 1px solid rgb(0, 0, 0); 
  background-color:white; 
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>borderColorButton</tabstop>
  <tabstop>bckgrndColorButton</tabstop>
  <tabstop>canvasColorButton</tabstop>
  <tabstop>borderWidthSpinBox</tabstop>
  <tabstop>titlesFontButton</tabstop>
  <tabstop>axesFontButton</tabstop>
  <tabstop>legendsFontButton</tabstop>
  <tabstop>treeCurve</tabstop>
  <tabstop>curveStyleCombo</tabstop>
  <tabstop>curveSymbolCombo</tabstop>
  <tabstop>lineStylecombo</tabstop>
  <tabstop>curveWidthSpinBox</tabstop>
  <tabstop>symbolSizeSpinBox</tabstop>
  <tabstop>curveColorButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>helpButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>plotPropertiesGUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>plotPropertiesGUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
